frontend/src/components/common/Admin/SortTablePopover.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import {
+  Button,
+  Button as ChakraButton,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  Radio,
+  Stack,
+  useDisclosure,
+  Text,
+  RadioGroup,
+} from "@chakra-ui/react";
+import { ChevronDownIcon } from "../icons";
+
+const SortTablePopover = (): React.ReactElement => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  return (
+    <>
+      <ChakraButton
+        onClick={onOpen}
+        rightIcon={<ChevronDownIcon />}
+        variant="secondary"
+      >
+        Sort
+      </ChakraButton>
+      <Modal isOpen={isOpen} onClose={onClose}>
```

Comment:
I think this component might work better as a [popover](https://chakra-ui.com/docs/components/popover). Right now the positioning of the modal relative to the `sort` button in the component library looks a bit off

Reply:
Hmm you're right, I'll trying using a popover instead